pyupgrade/_main.py

```diff
@@ -758,13 +763,13 @@ def _typed_class_replacement(
     return end, attrs
 
 
-def _fix_py36_plus(contents_text: str) -> str:
+def _fix_py36_plus(contents_text: str, *, min_version: Version) -> str:
     try:
         ast_obj = ast_parse(contents_text)
     except SyntaxError:
         return contents_text
 
-    visitor = FindPy36Plus()
+    visitor = FindPy36Plus(min_version=min_version)
```

Comment:
🤔  I think this might count as a boolean trap, but rewriting `Py36Plus` as `Py37Plus` just for a small difference struck me as duplicative

(of course, no offence taken if you want to close this to get your own fix out quickly)